spec/model/mapping_spec.cr

```diff
@@ -199,6 +239,21 @@ describe Jennifer::Model::Mapping do
           OneFieldModel.find!(id).id.should eq(id)
         end
       end
+
+      context "model has column aliases" do
+        it "correctly maps column aliases" do
+          a = Author.create(
+            name1: "Samply",
+            name2: "Examplary"
+          )
+
+          Author.
+            where { _first_name == "Samply" }.
```

Comment:
prefer putting multiline method call at the beginning of a new row

```crystal
Author
  .where
  # ...
```

Reply:
Thanks for pointing that out. I honestly did not know that this was possible in Crystal. I just was used to it coming from Ruby and never asked :D